src/formatter/renameField.ts

```diff
@@ -0,0 +1,28 @@
+import * as E from "fp-ts/Either";
+import * as O from "fp-ts/Option";
+import { modifyAt } from "fp-ts/Record";
+import { identity, pipe } from "fp-ts/function";
+
+const jsonParse: <T>(
+  text: string
+) => E.Either<Error, Record<string, T>> = E.tryCatchK(JSON.parse, E.toError);
+
+export const renameField = <T>(
+  input: string,
+  field: keyof Record<string, T>,
+  newField: T
```

Comment:
I am not completely sure we need a generic type here, i guess that talking only about renaming a field we may be safe enough using a `string`, what do you think? @drmarro @AleDore

Reply:
you're right @CrisTofani ,maybe we could enforce typings with `NonEmptyString`. BTW `input` must be generic with T type, so we can safely assert that `field` is related to the same object :)

Reply:
@AleDore I implemented `input` as a `string` in order to apply the JSON.parse function later. Should the renameField function accept a Javascript object as `input`? The function returns a `string` because I applied the JSON.stringify function to the modified object. Should the renameField function return a Javascript object? Thanks!

Reply:
yes the input is already decoded as a json object :)